kube/int/test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -ex
+
+echo "Image serve tests..."
+
+curl --fail -o /tmp/calexp_i6_out.fits "https://lsst-lsp-int.ncsa.illinois.edu/api/image/v1/DC_W13_Stripe82?ds=calexp&ra=37.644598&dec=0.104625&filter=r"
```

Comment:
I take it this is still testing the old imgserv, do you want to test SODA instead?